util/exec.go

```diff
@@ -23,6 +23,7 @@ import (
 // from the binary's stdout and stderr to this program's stdout and stderr.
 func Exec(bin string, args ...string) error {
 	cmd := exec.Command(bin, args...)
```

Comment:
How's [this](https://github.com/appc/acbuild/pull/27/files#diff-5601779b2e1aae16ccfdcd9bc9d17dc6R111)?

I also [updated run](https://github.com/appc/acbuild/pull/27/files#diff-e40b1e3cb0587929a3c786527046d69aR123) to have a similar check.